<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="editedView">
    <property name="geometry">
     <rect>
      <x>969</x>
      <y>10</y>
      <width>921</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="restore">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>760</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openNew">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>820</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveEdited">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>860</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save edited</string>
    </property>
   </widget>
   <widget class="QLabel" name="weightLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1020</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="weightView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>521</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="editedLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>620</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Edited</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="selectY">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>675</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="transform">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>720</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="originalView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>921</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xLabel">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>640</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yLabel">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>680</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="selectX">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>635</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="originalLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveGradient">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>900</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save weight</string>
    </property>
   </widget>
   <widget class="QLabel" name="vpLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>1020</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical Paths</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="vpView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>640</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveVPaths">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>940</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Vpaths</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="hpView">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>640</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="hpLabel">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>1020</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal Paths</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveHPaths">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>980</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Hpaths</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
